ertviz/controllers/response_controller.py

```diff
@@ -1,83 +1,113 @@
 import dash
+import json
+import pandas as pd
 from dash.dependencies import Input, Output, State
 from dash.exceptions import PreventUpdate
-from ertviz.data_loader import get_ensemble
+from ertviz.data_loader import get_ensemble_url
 from ertviz.ert_client import get_response
-from ertviz.models import EnsemblePlotModel, PlotModel
+from ertviz.models import ResponsePlotModel, PlotModel, EnsembleModel
 from ertviz.controllers import parse_url_query
 
 
-def _get_realizations_data(realizations, x_axis):
+obs_color = "rgb(176, 28, 52)"
```

Comment:
Should we put this in a stylesheet?

Reply:
Thought about it and It might be a good idea :) It should include most of the properties then.

Reply:
Let's create an issue about a dedicated `.css` file once this'd been merged.